pkg/ui/rule.go

```diff
@@ -9,6 +9,10 @@ import (
 	"regexp"
 	"sort"
 
+	thanosrule "github.com/improbable-eng/thanos/pkg/rule"
+
```

Comment:
goimports please :)

Reply:
fun fact. It is after goimports ! (: It's broken:  https://github.com/improbable-eng/thanos/issues/984

Reply:
Yeah I usually manually remove empty new  lines and it then works :D